### ISSUE TITLE
Support Android Jelly Bean (4.1 / 4.2 / 4.3)

### ISSUE THREAD

**Issue by pihug12** [y20k/transistor#38]
Can you please add support for Android Jelly Bean (4.1 / 4.2 / 4.3)? Or at least Android 4.3?

Thanks!

**Comment by y20k** [comment]
Hi there, 

Thank you for you suggestion! During a phase in which I was planning and prototyping I thought a lot about which API level I should support. 

My reasoning did not include the actual feature sets of each API. Maybe a mistake in hindsight. But just looking at the raw numbers ([Platform versions](http://developer.android.com/about/dashboards/index.html)) made me choose KitKat as the minimum SDK. 

As I said actual feature sets didn't factor in, since I was not too familiar with Android at that time. Just reading on developer blogs how hard it is to support older systems scared me from going back too far.

Anyway. Your post made me revisit my decision. Looking briefly at the actual changes KitKat introduced, I only found one feature: **Reading from [external storage](http://developer.android.com/about/versions/android-4.4.html#BehaviorStorage) without an explicit permission.** My app for now relies on that feature. On the other hand: There are definitely ways to carve out an exception for Jelly Bean devices.

I need to balance / weigh out
- the necessary effort to support Jelly Bean 
vs. 
- the likely decline in relevance that Jelly Bean has.

For now my decision is clear. I am not going to support Jelly Bean within the next couple of month. But I will have an eye on the development on the deployment numbers that Jelly Bean has. If it stays around 20% for prolonged time, I will revisit my decision again.

**TL;DR**: Support for Jelly Bean is possible, but not planned for the foreseeable future.
